espGarden/main/include/wifi_manager.h

```diff
@@ -30,44 +17,6 @@ static void event_handler(void* arg, esp_event_base_t event_base, int32_t event_
     }
 }
 
-esp_err_t wifi_manager_init(void) {
-    ESP_ERROR_CHECK(esp_netif_init());
-    wifi_event_group = xEventGroupCreate();
-    ESP_ERROR_CHECK(esp_event_loop_create_default());
-    sta_netif = esp_netif_create_default_wifi_sta();
-    assert(sta_netif);
-
-    wifi_init_config_t cfg = WIFI_INIT_CONFIG_DEFAULT();
-    ESP_ERROR_CHECK(esp_wifi_init(&cfg));
-    ESP_ERROR_CHECK(esp_event_handler_register(WIFI_EVENT, ESP_EVENT_ANY_ID, &event_handler, NULL));
-    ESP_ERROR_CHECK(esp_event_handler_register(IP_EVENT, IP_EVENT_STA_GOT_IP, &event_handler, NULL));
-    ESP_ERROR_CHECK(esp_wifi_set_storage(WIFI_STORAGE_RAM));
-
-    return ESP_OK;
-}
-
-esp_err_t wifi_manager_set_credentials(const char *ssid, const char *password) {
-    wifi_config_t wifi_config = {
-        .sta = {
-            .ssid = "",
-            .password = ""
-        },
-    };
-    strncpy((char *)wifi_config.sta.ssid, ssid, sizeof(wifi_config.sta.ssid));
-    strncpy((char *)wifi_config.sta.password, password, sizeof(wifi_config.sta.password));
-
-    ESP_LOGI(WIFI_LOG_TAG, "Setting WiFi configuration SSID %s...", wifi_config.sta.ssid);
-    ESP_ERROR_CHECK(esp_wifi_set_mode(WIFI_MODE_STA));
-    ESP_ERROR_CHECK(esp_wifi_set_config(WIFI_IF_STA, &wifi_config));
-
-    return ESP_OK;
-}
-
-esp_err_t wifi_manager_connect(void) {
-    ESP_ERROR_CHECK(esp_wifi_start());
-    return ESP_OK;
-}
-
 int32_t wifi_manager_get_rssi(void) {
```

Comment:
this function can likely get removed unless we plan sending to the webserver.

Reply:
I think its a cool parameter and will definitely be useful for logging purposes. Maybe we can see if there is a decrease in signal strength in the rain? Should we use a bigger antenna on the access point?

Reply:
ok Ya as long we have some purpose for it. hadn't looked past its usefulness other than for testing